definitions/ext-ddmysql/golden_metrics.yml

```diff
@@ -0,0 +1,1462 @@
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+mutexOSWaits:
+  title: Mutex OS Waits (event)
+  query:
+    select: mysql.innodb.mutex_os_waits
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+mutexSpinRounds:
+  title: Mutex Spin Rounds (event)
+  query:
+    select: mysql.innodb.mutex_spin_rounds
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+mutexSpinWaits:
+  title: Mutex Spin Waits (event)
+  query:
+    select: mysql.innodb.mutex_spin_waits
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+secondsBehindMaster:
+  title: Lag Between Master and Slave (second)
+  query:
+    select: mysql.replication.seconds_behind_master
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+slaveRunning:
+  title: Server is Replication Slave / Master running
+  query:
+    select: mysql.replication.slave_running
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+slavesConnected:
+  title: Slaves connected to replication master
+  query:
+    select: mysql.replication.slaves_connected
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+tableCacheHits:
+  title: Table Cache Hits (hit)
+  query:
+    select: mysql.performance.table_cache_hits
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+tableCacheMisses:
+  title: Table Cache Misses (miss)
+  query:
+    select: mysql.performance.table_cache_misses
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+xLockSpinRounds:
+  title: X Lock Spin Rounds
+  query:
+    select: mysql.innodb.x_lock_spin_rounds
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+wsrepLocalRecvQueueAvg:
+  #summary
+  title: Avg Size of Local Received Queue Since Last Status Query
+  query:
+    select: mysql.galera.wsrep_local_recv_queue_avg
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+wsrepFlowControlPaused:
+  title:  Time Since FLUSH STATUS Last Called, Paused Node due to Flow Control (fraction of time)
+  query:
+    select: mysql.galera.wsrep_flow_control_paused
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+wsrepFlowControlPausedNs:
+  title: Pause Time due to Flow Control (nanoseconds)
+  query:
+    select: mysql.galera.wsrep_flow_control_paused_ns
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+wsrepFlowControlRecv:
+  title: Galera Node Received Pausing Flow Control Message
+  query:
+    select: mysql.galera.wsrep_flow_control_recv
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+wsrepFlowControlSent:
+  title: Galera Node Sent Pausing Flow Control Message
+  query:
+    select: mysql.galera.wsrep_flow_control_sent
+    from: Metric
+    eventId: entity.guid
+    eventName: entity.name
+wsrepCertDepsDistance:
+  title: Average Distance Between Lowest and Highest SeqNo Node Can Apply in Parallel
```

Comment:
This is potentially a summary metric, but I wasn't sure since it is technically already averaged by the time it is given to us. (I'm new to all of the terminology and such so please correct me if I am misunderstanding anything :) )

Reply:
The difference will be in how they are displayed, being that summary metrics go to a table.

Reply:
So to make a recap:

Summary metrics are displayed in the explorer table.
Golden metrics can be displayed in a bunch of places. And we normally display up to 4 (but 10 is the real limit)

If you want to display what data we have for an entity in the entity view you would create a dashboard for that.

Reply:
Ah I see, thank you both!